source/expressions.tex

```diff
@@ -6584,19 +6584,21 @@
 A \grammarterm{throw-expression} is of type \tcode{void}.
 
 \pnum
-Evaluating a \grammarterm{throw-expression} with an operand throws an
-exception\iref{except.throw}; the type of the exception object is determined by removing
-any top-level \grammarterm{cv-qualifier}{s} from the static type of the
-operand and adjusting the type
-from ``array of \tcode{T}'' or function type \tcode{T}
-to ``pointer to \tcode{T}''.
+A \grammarterm{throw-expression} with an operand throws an exception\iref{except.throw}.
+The array-to-pointer\iref{conv.array} and function-to-pointer\iref{conv.func}
+standard conversions are performed on the operand.
+The type of the exception object\iref{except.throw} is determined by
+removing any top-level cv-qualifiers from the type of the (possibly-converted) operand.
 
 \pnum
 \indextext{exception handling!rethrow}%
-A
-\grammarterm{throw-expression}
+\indextext{exception handling!terminate called@\tcode{terminate} called}%
+\indextext{\idxcode{terminate}!called}%
+A \grammarterm{throw-expression}
 with no operand rethrows the currently handled exception\iref{except.handle}.
-The exception is reactivated with the existing exception object;
+If no exception is currently being handled,
+the function \tcode{std::terminate} is called\iref{except.terminate}.
```

Comment:
```suggestion
the function \tcode{std::terminate} is invoked\iref{except.terminate}.
```

Reply:
Generally-speaking, we consider "call" to be syntactic (`f(a,b)` is a function call) and "invoke" to be semantic (`a + b` for an overloaded `operator+` invokes a function but doesn't involve a function call).